op-e2e/actions/l2_sequencer_test.go

```diff
@@ -16,14 +20,30 @@ import (
 	"github.com/ethereum-optimism/optimism/op-service/testlog"
 )
 
+func EngineWithP2P() EngineOption {
+	return func(ethCfg *ethconfig.Config, nodeCfg *node.Config) error {
+		p2pKey, err := crypto.GenerateKey()
+		if err != nil {
+			return err
+		}
+		nodeCfg.P2P = p2p.Config{
+			MaxPeers:    100,
+			NoDiscovery: true,
+			ListenAddr:  "0.0.0.0:0",
```

Comment:
```suggestion
			ListenAddr:  "127.0.0.1:0",
```
To not bind them to external facing IP. Localhost should be enough